tools/logger/convert.c

```diff
@@ -891,9 +891,7 @@ static int logger_read(void)
 			 * report errors only for the DMA trace.
 			 */
 			if (global_config->trace && ldc_address_OK) {
-				/* FIXME: make this a log_err() */
-				fprintf(global_config->out_fd,
-					"warn: log_entry_address %#8x is not in dictionary range!\n",
+				log_err("log_entry_address %#8x is not in dictionary range!\n",
```

Comment:
Looks fine, why draft ?

Reply:
I _always_ start all my PRs as drafts. That avoids notifying people with silly checkpatch or other fixes (also saves even thinking about it). It works great except for this guy @lgirdwood who seems to use a different notification technique :-)